content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
+the details of the validation will be added to the chatter.
 
-.. image:: argentina/argentina_edi_18.png
+.. image:: argentina/afip-auth-rejected.png
    :align: center
-
+   :alt: AFIP authorization Rejected.
 
 Special Use cases
 ~~~~~~~~~~~~~~~~~
+
 Untaxed Concepts
 ****************
-There are some transactions that include items that are not part of the VAT base amount, 
-this is commonly used in fuel and gasoline invoices. 
 
-The vendor bill will be registered using 1 item for each product that is part of the VAT
-base amount and an additional item to register the amount of  the Exempt concept:
+There are some transactions that include items that are not part of the VAT base amount, this is
+commonly used in fuel and gasoline invoices. 
+
+The vendor bill will be registered using 1 item for each product that is part of the VAT base
+amount and an additional item to register the amount of the Exempt concept:
 
-.. image:: argentina/argentina_edi_19.png
+.. image:: argentina/vat-exempt.png
    :align: center
+   :alt: VAT exempt.
 
 Perception Taxes
 ****************
-The vendor bill will be registered using 1 item for each product that is part of the 
-VAT base amount, the perception tax can be added in any of the product lines, as result 
-we will have one tax group for the VAT and one for the perception, the perception default 
-value is always 1.00. 
 
-.. image:: argentina/argentina_edi_20.png
+The vendor bill will be registered using 1 item for each product that is part of the VAT base
+amount, the perception tax can be added in any of the product lines, as result we will have one
+tax group for the VAT and one for the perception, the perception default value is always
+:guilabel:`0.10`.
+
+.. image:: argentina/vat-perception.png
    :align: center
+   :alt: VAT perception.
 
-You should use the pencil that is the next to the Perception amount to edit it 
-and set the correct amount. 
+To edit it and set the correct amount you should use the :guilabel:`Pencil` that is the next to the
+Perception amount.
 
-.. image:: argentina/argentina_edi_21.png
+.. image:: argentina/enter-perception-amount.png
    :align: center
+   :alt: Enter the perception amount.
    
-After this is done the invoice can be validated. 
-   
+Then invoice can be validated.
    
 Reports
 =======
 
 As part of the localization the next Financial reports were added:
 
-.. image:: argentina/argentina18.png
+.. image:: argentina/argentinian-reports.png
    :align: center
+   :alt: Argentinian reports.
 
 VAT Reports
 -----------
 
-Libro de IVA Ventas
-~~~~~~~~~~~~~~~~~~~
+Sales VAT book
+~~~~~~~~~~~~~~
 
-.. image:: argentina/argentina19.png
+.. image:: argentina/sales-vat-book.png
    :align: center
+   :alt: Sales VAT book.
 
-Libro de IVA Compras
-~~~~~~~~~~~~~~~~~~~~
+The :guilabel:`Sales VAT` book report can be exported in a Zip file :guilabel:`VAT BOOK (ZIP)`
+button in the top left, which contains TXT files to upload in the AFIP portal.
```

Comment:
The text and the image need to switch places here.

As well, need more context in the writing:
- what is a Sales VAT book
- where is it located / how to navigate to it?
- why should a user care about it / what's it's purpose?